app/views/components/_markdown_editor.html.erb

```diff
@@ -23,23 +23,23 @@
       <div class="app-c-markdown-editor__toolbar">
         <markdown-toolbar class="app-c-markdown-editor__toolbar-group" for="<%= textarea[:id] %>">
           <% if controls.include?(:headings) %>
-            <md-header-2 class="app-c-markdown-editor__toolbar-button" title="Heading level 2" aria-label="Heading level 2" data-gtm="markdown-toolbar-h2" data-gtm-click-tracking="true">
+            <md-header-2 class="app-c-markdown-editor__toolbar-button" title="Heading level 2" aria-label="Heading level 2">
```

Comment:
How do we know this is for the old UA analytics?

Reply:
thats a good question.
I was confused about it too to begin with. My understanding is data-gtm tags are for old google tag manager where was new tags are data-ga4 for google analytics 4.
hence I understood this as old code and to be removed.